blocks/hero/hero.js

```diff
@@ -19,9 +19,9 @@ export function addBackLink(block, locale, placeholders, curPath) {
   // The 2 elements are created in the 'backward' order to allow the hover over the link
   // to change the color of the arrow. They are put in the correct position via the
   // 'order' attribute in the CSS.
-  bl.innerHTML = `<a href="/${locale}/${placeholders.newseventsbase}/"
+  bl.innerHTML = `<a href=${placeholders.herohref}
```

Comment:
Looks good. Only thing I'd want to change is rename `herohref` to something like `news-and-events-url` as likely this can be used at other places as well beyond just the `hero` block.